cdap-api-spark/src/main/scala/co/cask/cdap/api/spark/SparkExecutionContext.scala

```diff
@@ -118,6 +118,26 @@ trait SparkExecutionContext extends RuntimeContext with Transactional {
                                             splits: Option[Iterable[_ <: Split]]): RDD[(K, V)]
 
   /**
+    * Creates a [[org.apache.spark.rdd.RDD]] from the given [[co.cask.cdap.api.dataset.Dataset]].
+    * Using the implicit object [[co.cask.cdap.api.spark.SparkMain.SparkProgramContextFunctions]] is preferred.
+    *
+    * @param sc the [[org.apache.spark.SparkContext]] to use
+    * @param namespace namespace of the dataset
```

Comment:
in some places we say "namespace in which the stream exists" . In other places, just "namespace of the stream". Can you make it consistent throughout the Javadocs?